### PR TITLE
Quick grammar fixes in docs

### PR DESCRIPTION
**Pull request by agudulin**

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ const mockData = require('./mocks');
 <Message content={mockData.hello} />
 ```
 
-As an utility, also the [Lodash](https://lodash.com/) library is available globally as `_`.
+As a utility, also the [Lodash](https://lodash.com/) library is available globally as `_`.
```
Comment (sapegin) [inline docs/GettingStarted.md]: But why?

Comment (sapegin) [inline docs/GettingStarted.md]: Found already: https://www.englishforums.com/English/AUtilityVsAnUtility/hrmlw/post.htm

Comment (tizmagik) [inline docs/GettingStarted.md]: `a` vs `an` is determined by the _sound_ the word makes, not necessarily the letter that it starts with.

http://www.quickanddirtytips.com/education/grammar/a-versus-an

This is one of those English language oddities :)

Comment (agudulin) [inline docs/GettingStarted.md]: JFYI this wikihow helped me: http://www.wikihow.com/Use-%22A%22-and-%22An%22-Correctly

**Pull request merged**